### PR TITLE
Reinstating fix to #181 and addresses initial comment of #552

### PR DESCRIPTION
**Pull request by Str3am786**

### CODE REVIEW

File: src/somef/regular_expressions.py [hunk 1]
```diff
@@ -187,21 +187,44 @@ def extract_repo_status(unfiltered_text, repository_metadata: Result, readme_sou
...
         link = unfiltered_text[position:end]
         results.append(link.replace('arXiv:', 'https://arxiv.org/abs/'))
 
-    return results
+    for link in set(results):
+        repository_metadata.add_result(constants.CAT_RELATED_PAPERS,
+                                        {
+                                            constants.PROP_TYPE: constants.URL,
+                                            constants.PROP_VALUE: link,
+                                            constants.PROP_DESCRIPTION: arxiv_desc
```
Comment (dgarijo) [inline src/somef/regular_expressions.py:223]: Line 223 is wrong. You are adding an empty description. PLease delete it

**Comment by dgarijo** [comment]
@Str3am786 you are missing:
adding the category in the output.md in the specification (documentation). Also, adding the category in the readme. Can you please fix it?

**Comment by Str3am786** [comment]
New commit, fixing issues with unnecessary description and added 'related_papers' to readme and output.

**Comment by dgarijo** [comment]
@Str3am786 thanks.
Please push your changes to the `dev` branch, and I will merge them in master. See: https://github.com/KnowledgeCaptureAndDiscovery/somef#contribute

**Pull request closed without merging**